services/QuillLMS/client/app/bundles/Connect/components/studentLessons/lesson.jsx

```diff
@@ -525,6 +527,15 @@ export class Lesson extends React.Component {
           saveToLMS={this.saveToLMS}
         />
       );
+    } else if (languageOptions && hasTranslationFlag()) {
```

Comment:
Are we ever going to see the Register component this way? Maybe it doesn't matter or I'm reading it incorrectly, but it seems like we may never hit the `else` block if this is the case.

Reply:
that's a good catch-- I'll need to rework the logic a bit